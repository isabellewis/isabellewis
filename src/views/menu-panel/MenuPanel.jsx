import React from 'react';
import styled from 'styled-components';
import { Menu } from '../../components';

const MenuWrapper = styled.nav`
	background: #fff;
	opacity: .85;
	padding: 10px 15px;
	position: fixed;
	width: 100%;

	@media only screen and (min-width: 576px) {
		padding: 10px 20px;
	}
`;

export const MenuPanel = () => (
	<MenuWrapper>
		<Menu />
	</MenuWrapper>
);
