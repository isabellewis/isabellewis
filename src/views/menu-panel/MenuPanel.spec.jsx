import React from 'react';
import ReactDOM from 'react-dom';
import { shallow } from 'enzyme';
import { MenuPanel } from './MenuPanel';

describe('views/MenuPanel', () => {
	let component;

	beforeEach(() => {
		component = shallow(<MenuPanel />);
	});

	it('should match the snapshot', () => {
		expect(component).toMatchSnapshot();
	});
});