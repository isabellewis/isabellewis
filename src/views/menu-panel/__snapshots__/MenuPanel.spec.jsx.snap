// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`views/MenuPanel should match the snapshot 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <MenuPanel />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "function",
    "props": Object {
      "children": <Menu />,
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {},
      "ref": null,
      "rendered": null,
      "type": [Function],
    },
    "type": Object {
      "$$typeof": Symbol(react.forward_ref),
      "attrs": undefined,
      "componentStyle": ComponentStyle {
        "componentId": "MenuPanel__MenuWrapper-jetzIn",
        "isStatic": true,
        "rules": Array [
          "background:#fff;opacity:.85;padding:10px 15px;width:100%;@media only screen and (min-width:576px){padding:10px 40px 10px 20px;}",
        ],
      },
      "displayName": "MenuPanel__MenuWrapper",
      "render": [Function],
      "styledComponentId": "MenuPanel__MenuWrapper-jetzIn",
      "target": "nav",
      "toString": [Function],
      "warnTooManyClasses": [Function],
      "withComponent": [Function],
    },
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "function",
      "props": Object {
        "children": <Menu />,
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {},
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      "type": Object {
        "$$typeof": Symbol(react.forward_ref),
        "attrs": undefined,
        "componentStyle": ComponentStyle {
          "componentId": "MenuPanel__MenuWrapper-jetzIn",
          "isStatic": true,
          "rules": Array [
            "background:#fff;opacity:.85;padding:10px 15px;width:100%;@media only screen and (min-width:576px){padding:10px 40px 10px 20px;}",
          ],
        },
        "displayName": "MenuPanel__MenuWrapper",
        "render": [Function],
        "styledComponentId": "MenuPanel__MenuWrapper-jetzIn",
        "target": "nav",
        "toString": [Function],
        "warnTooManyClasses": [Function],
        "withComponent": [Function],
      },
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
